docs/spec.md

```diff
@@ -61,6 +61,7 @@ Each JSON message must be an object containing a field called `op` which identif
   - `parametersSubscribe`: Allow clients to subscribe to parameter changes
   - `time`: The server may publish binary [time](#time) messages
 - `supportedEncodings`: array of strings | informing the client about which encodings may be used for client-side publishing. Only set if client publishing is supported.
+- `options`: optional map of key-value pairs
```

Comment:
Hm, maybe call this `metadata` to mirror mcap channel metadata? What do you think @jhurliman?  cc @james-rms

Reply:
I agree, it sounds better

```suggestion
- `metadata`: optional map of key-value pairs
```